src/fsearch_window.c

```diff
@@ -725,6 +725,36 @@ on_listview_row_is_selected(int row, gpointer user_data) {
     return FALSE;
 }
 
+static GList*
+listview_on_drag_data_get(GList *indices, gpointer user_data) {
```

Comment:
You don't need to pass the indices of all the selected entries here. Instead you can simply call `db_view_selection_for_each(db_view, some_prepend_path_helper, &uris);` here to collect all the selected paths.

Reply:
Or even better, instead of having `some_prepend_path_helper()` building a new GList, use it to build a string array right away, as that's what we need for `gtk_selection_data_set_uris()`.